### PR TITLE
fix: add async and creatable to SelectInputProps

### PR DESCRIPTION
**Pull request by yurii-korolchuk**
Just tried to use Select with creatable prop, and turns out we don't have it in SelectInputProps. Or do I miss something ?

### CODE REVIEW

File: src/components/select/index.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import ChipsComponent from './chips-component';
 import SelectComponent from './select-component';
 
 export interface SelectInputProps extends SelectProps, InputComponent {
```
Comment (KotAleksei) [inline src/components/select/index.tsx:10]: we are using SelectInputProps and ComponentProps in the './select-components.tsx' file, these interfaces are almost the same and extend the same interfaces (SeleProps, InputComponent), let's rewrite it a bit and extend types from there

```
import type { ComponentProps } from './select-component';

export interface SelectInputProps extends ComponentProps {
  clearChips?: boolean;
  hideChips?: boolean;
}
```

**Review by KotAleksei — CHANGES_REQUESTED** [review]

**Review by vozbrann — APPROVED** [review]

File: src/components/select/index.tsx [hunk 2]
```diff
@@ -1,27 +1,20 @@
...
-    sm?: number | string;
-  };
   hideChips?: boolean;
 }
 
 export const SelectInput = React.forwardRef<HTMLDivElement, SelectInputProps>(
   (
     {
+      async,
+      creatable,
```
Comment (vlacher12) [inline src/components/select/index.tsx:17]: @yurii-korolchuk What's the purpose of taking out async and creatable from `...props`?  `async` and `creatable` are not used here. You pass `...props` in `SelectComponent` already.

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/select/index.tsx (hunk 2 above)
Comment (yurii-korolchuk) [reply src/components/select/index.tsx]: @vlacher12 removed

**Comment by vlacher12** [comment]
@yurii-korolchuk Does it still work in story?

**Comment by yurii-korolchuk** [comment]
@vlacher12 haha, it doesn't and it doesn't work on master branch either, I'll fix it right now

**Comment by yurii-korolchuk** [comment]
@vlacher12 done

**Review by vlacher12 — APPROVED** [review]

**Pull request merged**